deltacat/catalog/model/catalog.py

```diff
@@ -16,9 +16,7 @@
 from deltacat.catalog.iceberg import IcebergCatalogConfig
 from deltacat.constants import DEFAULT_CATALOG
 
-# registry of namespace to ActorHandle for Catalogs class
-catalog_registry: Dict[str, ray.actor.ActorHandle] = {}
-_default_namespace = "DEFAULT"
```

Comment:
I'm having trouble understanding the use case of Ray Namespace in DeltaCAT (but since we are removing that, and the code doesn't seem to show up in many places, which probably makes it not so important.) Will ask during 4/9 PythonSDK standup parking lot.

Will approve for now to not block code merge